### PR TITLE
Game screen navigation, also updated navbar

### PR DESCRIPTION
**Pull request by beckskelly**
Aware its the weekend so review this at your leisure/monday

Set up initial page for Game Screen and changed navbar

1. Navbar now shows you when youre on an active page, moved username to right hand side also. Put "Game" in front of stocks
2. Game screen has navigation for 4 diff elements on same screen, have taken out portfolio page from its old route and put in here
3. Deleted portfolio page route and threw it into the game

you can view game screen on /game/gamescreen, aware this is not the correct route but wasnt sure what to do so put it there for now

image

image

### CODE REVIEW

File: src/components/gameComponents/individualGameScreenComponents/gameNavigation/GameNavBar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Navbar, Container, Nav } from 'react-bootstrap'
+
+function GameNavBar({ disPlayScreen, active }) {
+
+    return (
+        <Navbar expand="lg" className="navbar navbar-expand" variant="dark">
+
+            <Container>
+                <Nav className="me-auto" >
+                    {/* <Nav.Link style={{ color: 'white' }}>Activity</Nav.Link> */}
```
Comment (joeycor92) [inline src/components/gameComponents/individualGameScreenComponents/gameNavigation/GameNavBar.js:10]: Might remove this.

**Review by joeycor92 — CHANGES_REQUESTED** [review]
It looks good. Just two things, should we have it showing which is the active page at the top, i.e. games and not stocks? 

The other thing was just to remove the commented out line.

**Comment by beckskelly** [comment]
Sound @joeycor92, I got rid of the line. I think it will show the active page properly when we decide on a proper route for it (i.e. link it up to the backend), and have a proper flow to the screen. If you try to create a game on the branch, youll see its highlighted properly there

**Review by joeycor92 — APPROVED** [review]
Sounds great, happy to approve now.

**Review by WarrenK-design — CHANGES_REQUESTED** [review]
Hi Becks, 

The game screen and nav bar look good. 

Do you mind if you keep the portfolio screen for the moment, me and Daragh where talking last week and we may still be able to use this with the dropdown menu with the routes hes creating for a general "Portfolio" view. 

For the individual GameScreen though can you get rid of this dropdown, from the game screen it should only show them there portfolio associated with that game. 
image

As well could you add a route for ```/game/:gameId``` for this screen so that when I click on these cards it brings me to this game Screen. 
image

Just small thing too that catches eye card doesn't line up. 
image

**Comment by beckskelly** [comment]
Hey @WarrenK-design , thanks for the review, some good points made here, so I made the following changes.

1. fixed the allignment for the portfolio screen
2. Got rid of dropdown in game version
3. Added back in the portfolio route, its in /portfolio
4. Added a link to the game cards and updated the route to /game/:gameId so theyre connected now
5. Got rid of those errors too

I understand you've been a bit held up in reviews the day, so feel free to ignore this until tomorrow

**Review by WarrenK-design — APPROVED** [review]
Looks good Becks cheers for that!

**Pull request merged**